dukechapel/views/digital_collections/about.html.erb

```diff
@@ -16,7 +16,7 @@
 
     <p>For their work on the creation of the initial digital collection, special thanks to the Rev. Dr. Charles Campbell, Professor of Homiletics at Duke Divinity School, the Rev. Dr. Luke Powery, Dean of Duke Chapel; the Rev. Dr. Fred Westbrook, C’Access Inc.; and Adrienne Koch, Communications Specialist, Duke Chapel and Project Director. Special thanks also to Duke Divinity graduate, Ben Vockery, who fully inventoried the collection of recordings in preparation for digitization.</p>
 
-    <p>For their work on the Lilly Endowment phase of the collection, special thanks to the Rev. Drs. Luke Powery and Chuck Campbell, who spearheaded the grant; Jessica Serrao, University Archives intern, who coordinated the digitization phase; Divinity student, Katie Murchison Ross, who coordinated permissions; and Adrienne Koch, the project director of the transcription and metadata phases.</p>
+    <p>For their work on the Lilly Endowment phase of the collection, special thanks to the Rev. Drs. Luke Powery and Chuck Campbell, who spearheaded the grant; Jessica Serrao, William King intern, who coordinated the digitization phase; Divinity student, Katie Murchison Ross, who coordinated permissions; and Adrienne Koch, the project director of the transcription and metadata phases.</p>
```

Comment:
@corylown No, though it's definitely confusing; "William King intern" is actually a position name here. William King is/was a former Duke University Archivist.